app/assets/javascripts/discourse/app/widgets/search-menu-results.js

```diff
@@ -819,7 +871,7 @@ createWidget("random-quick-tip", {
 createWidget("search-menu-recent-searches", {
   tagName: "div.search-menu-recent",
 
-  template: hbs`
+  template: widgetHbs`
```

Comment:
why do we need `widgetHbs` here?

Reply:
B/c this import is named `hbs` and to avoid conflict I am importing the default here to `widgetHbs` instead. They are indeed 2 different imports.